mlir/test/Dialect/EmitC/invalid_ops.mlir

```diff
@@ -137,14 +137,6 @@ func.func @cast_tensor(%arg : tensor<f32>) {
 
 // -----
 
-func.func @cast_array(%arg : !emitc.array<4xf32>) {
-    // expected-error @+1 {{'emitc.cast' op operand type '!emitc.array<4xf32>' and result type '!emitc.array<4xf32>' are cast incompatible}}
-    %1 = emitc.cast %arg: !emitc.array<4xf32> to !emitc.array<4xf32>
```

Comment:
Shouldn't this still be invalid? This is checked in the `verify`, right?

Reply:
Yes, indeed. Thanks!